src/client/dune

```diff
@@ -2,7 +2,7 @@
   (name        opam_client)
   (public_name opam-client)
   (synopsis    "OCaml Package Manager client and CLI library")
-  (modules     (:standard \ opamMain opamManifest get-git-version))
+  (modules     (:standard \ opamMain opamManifest get_git_version))
```

Comment:
As this script was being altered, I pulled in the change from https://github.com/ocaml/opam/pull/4096.